lib/styled.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+let cli   = require('..');
+
+String.prototype.toTitleCase = function() {
+  return this.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
+};
+
+String.prototype.ljust = function( width, padding ) {
+	padding = padding || " ";
+	padding = padding.substr( 0, 1 );
+	if( this.length < width )
+		return this + padding.repeat( width - this.length );
+	else
+		return this;
+};
+
+Array.prototype.max = function() {
+  return Math.max.apply(null, this);
+};
+
+Object.prototype.keys = function ()
+{
+  var keys = [];
+  for(var i in this) if (this.hasOwnProperty(i))
+  {
+    keys.push(i);
+  }
+  return keys;
+};
+
+/**
+ * styled_header logs in a consistent header style
+ *
+ * @example
+ * styled_header('MyApp') # Outputs === MyApp
+ *
+ * @param {header} header text
+ * @returns {null}
+ */
+function styled_header(header) {
+  cli.log(`=== ${header}`);
```

Comment:
this is already in `lib/header.js`. It might make sense here now, but it should be consolidated.